docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md

```diff
@@ -1,11 +1,12 @@
-# Themes
+# Themes and UI Replacements
 
-A list of known working qBittorrent themes.
+A list of known working qBittorrent themes and UI replacements.
 
-| Theme                                                                                                                           | Style | Compatibility   |
-| ------------------------------------------------------------------------------------------------------------------------------- | ----- | --------------- |
-| [Honeywell](https://github.com/stacksmash76/qbt-honeywell){:target="\_blank" rel="noopener noreferrer"}                         | Dark  | v4.3.1 - v4.4.5 |
-| [Nightwalker Theme (CallMeBruce fork)](https://github.com/CallMeBruce/nightwalker){:target="\_blank" rel="noopener noreferrer"} | Dark  | v4.5            |
-| [World of Quinoa](https://github.com/gl0ryus/woq){:target="\_blank" rel="noopener noreferrer"}                                  | Dark  | v4.3.9, v4.4.5  |
+| Theme                                                                                                                           | Type           | Style      | Compatibility   |
+| ------------------------------------------------------------------------------------------------------------------------------- | -------------- | ---------- | --------------- |
+| [Honeywell](https://github.com/stacksmash76/qbt-honeywell){:target="\_blank" rel="noopener noreferrer"}                         | Theme          | Dark       | v4.3.1 - v4.4.5 |
```

Comment:
THis URL isn't valid anymore, and I can't find the replacement URL

Reply:
Looks like the repo has been deleted - I've removed it.